### PR TITLE
nfs-ganesha: improvements to the active-active deployment section

### PR DESCRIPTION
**Pull request by rjfd**
Signed-off-by: Ricardo Dias <rdias@suse.com>

### CODE REVIEW

File: xml/deployment_ganesha.xml [hunk 1]
```diff
@@ -504,10 +504,9 @@ cephfs_metadata
    <title>Configure &ganesha;</title>
    <para>
     Check that both &ganesha; nodes have the file
-    <filename>/etc/ganesha/ganesha.conf</filename> files installed. The
-    <package>nfs-ganesha-ceph</package> package ships with a sample
-    <filename>/etc/ganesha/ceph.conf</filename> file that you can tweak as
-    needed. Following is an example of <filename>ceph.conf</filename>:
+    <filename>/etc/ganesha/ganesha.conf</filename> files installed. Add the
+    following blocks, if they not exist yet, to the configuration file in
+    order to enable RADOS as the recovery backend of &ganesha.
```
Comment (asettle) [inline xml/deployment_ganesha.xml]:
```suggestion
    order to enable RADOS as the recovery backend of &ganesha;.

Comment (rjfd) [reply xml/deployment_ganesha.xml]: done

File: xml/deployment_ganesha.xml [hunk 2]
```diff
@@ -526,26 +525,26 @@ CACHEINODE {
...
+    UserId = "<replaceable>cephx_user_id</replaceable>";
+    Ceph_Conf = "<replaceable>path_to_ceph.conf</replaceable>"
 }
 </screen>
+   <para>
+   You can find out the values for <replaceable>rados_pool</replaceable> and
+   <replaceable>pool_namespace</replaceable> by checking the already existing line in
+   the configuration of the form
+   <screen>%url rados://<replaceable>rados_pool</replaceable>/<replaceable>pool_namespace</replaceable>/...
+   </screen>.
```
Comment (asettle) [inline xml/deployment_ganesha.xml]: Sorry, I didn't see this before. It's complaing about this. You can't have a screen tag within a <para>.
So close the para after "form" and open again when you start with "The value"

Comment (rjfd) [reply xml/deployment_ganesha.xml]: Thanks for the help. Pushed the fix

**Comment by asettle** [comment]
Great! Thanks. Requesting one more reviewer :)

File: xml/deployment_ganesha.xml [hunk 3]
```diff
@@ -504,10 +504,9 @@ cephfs_metadata
    <title>Configure &ganesha;</title>
    <para>
     Check that both &ganesha; nodes have the file
-    <filename>/etc/ganesha/ganesha.conf</filename> files installed. The
-    <package>nfs-ganesha-ceph</package> package ships with a sample
-    <filename>/etc/ganesha/ceph.conf</filename> file that you can tweak as
-    needed. Following is an example of <filename>ceph.conf</filename>:
+    <filename>/etc/ganesha/ganesha.conf</filename> files installed. Add the
```
Comment (tbazant) [inline xml/deployment_ganesha.xml]:
```suggestion
    <filename>/etc/ganesha/ganesha.conf</filename> installed. Add the
```

File: xml/deployment_ganesha.xml [hunk 4]
```diff
@@ -504,10 +504,9 @@ cephfs_metadata
...
+    following blocks, if they not exist yet, to the configuration file in
```
Comment (tbazant) [inline xml/deployment_ganesha.xml]:
```suggestion
    following blocks, if they do not exist yet, to the configuration file in
```

**Comment by rjfd** [comment]
@tbazant pushed the fixes

**Comment by asettle** [comment]
@rjfd - this is for SES 6?

**Comment by rjfd** [comment]
@asettle yes

**Comment by asettle** [comment]
> @asettle yes

Great, we'll just make sure this gets backported after merge. Awaiting @tbazant 's review.

**Pull request merged**